docs/design_decisions.md

```diff
@@ -1,203 +1,245 @@
 
-*Ratio*
-Zolang we niets doen met de abstracte types (subject, persoon, natuurlijk persoon), heeft het geen zin dit mee te nemen in de component schema's.
+_**Ratio**_
 
-*Kanttekening*
-Deze design decision is al lang geleden gemaakt. Meer recent hebben we uit technische hergebruik overwegingen abstracte types geïntroduceerd. Het is het overwegen waard om die technische keuzes te vergelijken met de keuzes uit de informatiemodellen en te controleren of daar uniformereing mogelijk en wenselijk is.
-
-## Sortering voor actuele zoekresultaten worden niet gesorteerd
-De API standaard schrijft niet voor hoe zoekresultaten in de API moeten worden gesorteerd. Wanneer de client behoefte heeft aan gesorteerde resultaten, moet zij de ontvangen resultaten zelf sorteren.
-
-## Historie wordt gesorteerd op geldigheid met meest actuele resultaat bovenaan
-Historie wordt aflopend gesorteerd op datum geldigheid (geldigVan).
-
-## Historie elementen krijgen dezelfde naam en betekenis als in de NEN3610
-beginGeldigheid en eindGeldigheid
-Binnen de NEN standaard is er een keuze-mogelijkheid (datum en datumtijd) voor het formaat waarin de historie wordt bijgehouden. De Bron houdt geen tijd bij dus schrijven we het datum-formaat voor.
+Zolang we niets doen met de abstracte types (subject, persoon, natuurlijk persoon), heeft het geen zin dit mee te nemen in de component schema's.
 
-## Historie: inonderzoek wordt alleen actueel getoond
-Binnen de historie-endpoints wordt alleen de actuele situatie met betrekking tot "in Onderzoek" getoond. Er wordt geen historie getoond van de onderzoeken die in het verleden hebben plaatsgevonden.  
+_**Kanttekening**_
 
-## Toepassen van HAL-Links
-Er zijn grofweg twee categoriën Hal-links waar we gebruik van maken. Links naar resources binnen het eigen domein en links naar resources die in een ander domein beheerd worden. Om discoverability te bereiken, worden voor beide categorieën de Hal-link opgenomen naar de gerelateerde resource.
+Deze design decision is al lang geleden gemaakt. Meer recent hebben we uit technische hergebruik overwegingen abstracte types geïntroduceerd. Het is het overwegen waard om die technische keuzes te vergelijken met de keuzes uit de informatiemodellen en te controleren of daar uniformering mogelijk en wenselijk is. 
 
-## Identificatie van de gerelateerde resources worden opgenomen in de content van de opgevraagde resource
-Voor developers die geen HAL links willen gebruiken wordt tevens de identificatie van de gerelateerde resource opgenomen.
+### DD5.9 Actuele zoekresultaten worden niet gesorteerd
+De API standaard schrijft niet voor hoe zoekresultaten in de API moeten worden gesorteerd. Wanneer de client behoefte heeft aan gesorteerde resultaten, moet zij de ontvangen resultaten zelf sorteren.
+Dit betekent dat er in de berichtspecificaties geen gebruik gemaakt wordt van de _'sorteer'_ parameter.
```

Comment:
Er wordt alleen een parameter sorteer opgenomen, wanneer daar behoefte aan is (die blijkt uit een user story). Maar dat geldt voor elke parameter.